src/utils.js

```diff
@@ -155,7 +160,7 @@ export const isDocker = (forceReset) => {
 /**
  * Sums an array of numbers.
  *
- * @param {Array} arr An array of numbers.
+ * @param {Array<Number>} arr An array of numbers.
```

Comment:
Would `number[]` notation work?

Reply:
yes, it understands `[]` and `Array<>` notations

Reply:
Then please use `number[]` notation everywhere @yin .

Reply:
Done. Just a question: Is there a difference between `Array<Number|String>` and `Number[]|String[]`. The former seems to be an array of mixed elements.